test/amount.test.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash')
+const { loadFixture } = require('./utils')
+const { coreTypes } = require('../dist/types')
+const { Amount } = coreTypes
+const fixtures = loadFixture('data-driven-tests.json')
+
+function amountErrorTests () {
+  _.filter(fixtures.values_tests, { type: 'Amount' }).forEach(f => {
+    // We only want these with errors
+    if (!f.error) {
+      return
+    }
+    const testName = `${JSON.stringify(f.test_json)}\n\tis invalid ` +
+                     `because: ${f.error}`
+    it(testName, () => {
+      expect(() => {
+        Amount.from(f.test_json)
+        JSON.stringify(f.test_json)
+      }).toThrow()
+    })
+  })
+}
+
+describe('Amount', function () {
+  it('can be parsed from', function () {
+    expect(Amount.from('1000000') instanceof Amount).toBe(true)
```

Comment:
changed `assert(...)` to `expect(...).toBe(true)` everywhere `assert(...)` was present